.travis.yml

```diff
@@ -49,3 +53,8 @@ jobs:
         - export ARCH=arm64 CROSS_COMPILE=/usr/bin/aarch64-linux-gnu-
         - bash kconfig/kconfig-sof-arm64.sh
         - KCFLAGS="-Wall -Werror" make -j`getconf _NPROCESSORS_ONLN`
+    - name: "BUILD CLANG SOF Kernel x86_64"
```

Comment:
@marc-hb But there is gcc in other names, right? And clang is whatever version kbuild team has packaged, so maybe better to leave that out as well (as clang version could be bumped without us touching travis yml).

Reply:
> But there is gcc in other names, right?

Now there is :-)

> And clang is whatever version kbuild team has packaged, so maybe better to leave that out as well (as clang version could be bumped without us touching travis yml).

Indeed  https://download.01.org/0day-ci/cross-package/ has only one clang version uploaded a couple weeks ago and called... `latest`! 

Pierre:
> I'd prefer it if we keep the same scripts as kbuild, that way I can reproduce the results locally. I don't use bionic

While there's still value in this PR forget about the reproducibility. This new Travis job should be called "clang unknown version of the day".

Reply:
> While there's still value in this PR forget about the reproducibility. This new Travis job should be called "clang unknown version of the day".

well, we've moved from the known to be obsolete compiler for years.
I actually don't mind if we have a test that tries the latest compiler. That would show errors that we didn't see before, without requiring an update on our side. It could be a feature.

For device tests I think we do need to have stability to avoid random variables introduced in the mix. But for pure compilation tests, it's not so bad to use the latest compiler